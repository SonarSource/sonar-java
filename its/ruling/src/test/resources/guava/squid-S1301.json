{
'com.google.guava:guava:src/com/google/common/cache/CacheBuilderSpec.java':[
184,
],
}
