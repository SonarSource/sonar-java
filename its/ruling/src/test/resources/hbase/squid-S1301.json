{
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AccessControlProtos.java':[
6159,
7058,
9587,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AdminProtos.java':[
2815,
7168,
11117,
12213,
15651,
17649,
17987,
19403,
19741,
21291,
21629,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/AuthenticationProtos.java':[
861,
2372,
3259,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/ComparatorProtos.java':[
3510,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/FilterProtos.java':[
6421,
15588,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/HBaseProtos.java':[
14221,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MasterProtos.java':[
1011,
2191,
3487,
4599,
5815,
6714,
7703,
8602,
17014,
17869,
18768,
20184,
23719,
24057,
24395,
24733,
25071,
26817,
27597,
29245,
31028,
32648,
33547,
41647,
42546,
48521,
50423,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/MultiRowMutationProtos.java':[
53,
391,
1633,
1979,
1995,
2011,
2051,
2070,
2086,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/QuotaProtos.java':[
231,
3981,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/RegionServerStatusProtos.java':[
2751,
3931,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/RowProcessorProtos.java':[
1461,
1477,
1493,
1533,
1552,
1568,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/SecureBulkLoadProtos.java':[
4146,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/WALProtos.java':[
11487,
],
}
