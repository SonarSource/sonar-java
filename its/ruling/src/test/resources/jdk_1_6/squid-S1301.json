{
'jdk_1_6:java/awt/Component.java':[
6177,
6333,
6376,
6416,
6455,
6491,
],
'jdk_1_6:java/awt/Container.java':[
2063,
],
'jdk_1_6:java/awt/TextComponent.java':[
685,
],
'jdk_1_6:java/awt/Window.java':[
1945,
],
'jdk_1_6:java/awt/event/ActionEvent.java':[
234,
],
'jdk_1_6:java/awt/event/AdjustmentEvent.java':[
209,
],
'jdk_1_6:java/awt/event/InvocationEvent.java':[
262,
],
'jdk_1_6:java/awt/event/ItemEvent.java':[
148,
],
'jdk_1_6:java/awt/event/TextEvent.java':[
82,
],
'jdk_1_6:java/awt/image/Raster.java':[
889,
978,
],
'jdk_1_6:java/io/LineNumberInputStream.java':[
82,
],
'jdk_1_6:java/io/LineNumberReader.java':[
114,
156,
],
'jdk_1_6:java/lang/CharacterDataLatin1.java':[
241,
],
'jdk_1_6:java/security/MessageDigest.java':[
379,
],
'jdk_1_6:java/text/RBTableBuilder.java':[
110,
],
'jdk_1_6:java/util/JapaneseImperialCalendar.java':[
1082,
1135,
],
'jdk_1_6:java/util/SimpleTimeZone.java':[
1507,
1521,
],
'jdk_1_6:java/util/regex/Pattern.java':[
2369,
],
}
