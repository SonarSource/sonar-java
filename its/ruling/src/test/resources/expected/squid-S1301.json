{
'project:com/sun/corba/se/PortableActivationIDL/_InitialNameServiceImplBase.java':[
39,
],
'project:com/sun/corba/se/PortableActivationIDL/_ORBProxyImplBase.java':[
38,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImpl.java':[
282,
1211,
1220,
],
'project:com/sun/corba/se/impl/dynamicany/DynAnyUtil.java':[
315,
],
'project:com/sun/corba/se/impl/interceptors/ClientRequestInfoImpl.java':[
944,
946,
],
'project:com/sun/corba/se/impl/protocol/CorbaMessageMediatorImpl.java':[
1950,
],
'project:com/sun/corba/se/spi/activation/_InitialNameServiceImplBase.java':[
35,
],
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
1255,
],
'project:com/sun/imageio/plugins/common/BogusColorSpace.java':[
49,
],
'project:com/sun/imageio/plugins/jpeg/SOFMarkerSegment.java':[
182,
],
'project:com/sun/imageio/plugins/png/PNGImageWriter.java':[
1105,
],
'project:com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java':[
127,
],
'project:com/sun/java/swing/plaf/windows/WindowsFileChooserUI.java':[
361,
],
'project:com/sun/java/swing/plaf/windows/WindowsScrollBarUI.java':[
345,
],
'project:com/sun/java/swing/plaf/windows/WindowsTableHeaderUI.java':[
130,
],
'project:com/sun/jmx/snmp/IPAcl/Parser.java':[
43,
51,
59,
105,
179,
276,
314,
372,
401,
447,
476,
531,
574,
637,
727,
768,
796,
890,
],
'project:com/sun/jmx/snmp/IPAcl/ParserTokenManager.java':[
692,
710,
],
'project:com/sun/jmx/snmp/IPAcl/Token.java':[
101,
],
'project:com/sun/jmx/snmp/agent/SnmpMibAgent.java':[
719,
],
'project:com/sun/jmx/snmp/daemon/CommunicatorServer.java':[
255,
],
'project:com/sun/org/apache/xerces/internal/dom/NodeImpl.java':[
1064,
],
'project:com/sun/org/apache/xerces/internal/impl/PropertyManager.java':[
61,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentFragmentScannerImpl.java':[
2601,
2645,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentScannerImpl.java':[
837,
1354,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/identity/XPathMatcher.java':[
460,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
1944,
],
'project:com/sun/org/apache/xml/internal/security/encryption/XMLCipher.java':[
1637,
1695,
2636,
],
'project:com/sun/org/apache/xml/internal/security/signature/XMLSignatureInputDebugger.java':[
609,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformBase64Decode.java':[
173,
],
'project:java/awt/Component.java':[
6392,
6548,
6591,
6631,
6670,
6706,
],
'project:java/awt/Container.java':[
2253,
],
'project:java/awt/TextComponent.java':[
701,
],
'project:java/awt/Window.java':[
2126,
],
'project:java/awt/event/ActionEvent.java':[
278,
],
'project:java/awt/event/AdjustmentEvent.java':[
257,
],
'project:java/awt/event/InvocationEvent.java':[
341,
],
'project:java/awt/event/ItemEvent.java':[
180,
],
'project:java/awt/event/TextEvent.java':[
104,
],
'project:java/awt/image/Raster.java':[
895,
981,
],
'project:java/io/LineNumberInputStream.java':[
99,
],
'project:java/io/LineNumberReader.java':[
131,
173,
],
'project:java/lang/CharacterDataLatin1.java':[
231,
],
'project:java/lang/invoke/MethodHandleNatives.java':[
338,
],
'project:java/lang/invoke/MethodTypeForm.java':[
411,
],
'project:java/nio/file/FileTreeWalker.java':[
52,
],
'project:java/security/MessageDigest.java':[
409,
],
'project:java/text/RBTableBuilder.java':[
128,
],
'project:java/util/JapaneseImperialCalendar.java':[
1100,
1153,
],
'project:java/util/Locale.java':[
829,
],
'project:java/util/SimpleTimeZone.java':[
1535,
1549,
],
'project:java/util/regex/Pattern.java':[
2608,
],
'project:javax/management/BinaryOpValueExp.java':[
154,
],
'project:javax/management/ObjectName.java':[
590,
938,
],
'project:javax/security/auth/Subject.java':[
1083,
],
'project:javax/swing/JTable.java':[
7149,
],
'project:javax/swing/SwingUtilities.java':[
866,
877,
],
'project:javax/swing/plaf/basic/BasicComboBoxUI.java':[
1055,
],
'project:javax/swing/plaf/basic/BasicScrollBarUI.java':[
239,
860,
1118,
1131,
1148,
1312,
],
'project:javax/swing/plaf/basic/BasicSliderUI.java':[
1615,
1637,
1753,
],
'project:javax/swing/plaf/basic/BasicTextFieldUI.java':[
254,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
342,
],
'project:javax/swing/plaf/nimbus/ArrowButtonPainter.java':[
75,
],
'project:javax/swing/plaf/nimbus/DesktopIconPainter.java':[
76,
],
'project:javax/swing/plaf/nimbus/DesktopPanePainter.java':[
85,
],
'project:javax/swing/plaf/nimbus/InternalFramePainter.java':[
90,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePanePainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/MenuBarMenuPainter.java':[
74,
],
'project:javax/swing/plaf/nimbus/MenuBarPainter.java':[
77,
],
'project:javax/swing/plaf/nimbus/MenuItemPainter.java':[
74,
],
'project:javax/swing/plaf/nimbus/OptionPaneMessageAreaOptionPaneLabelPainter.java':[
72,
],
'project:javax/swing/plaf/nimbus/OptionPaneMessageAreaPainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/PopupMenuPainter.java':[
75,
],
'project:javax/swing/plaf/nimbus/PopupMenuSeparatorPainter.java':[
72,
],
'project:javax/swing/plaf/nimbus/ScrollBarPainter.java':[
70,
],
'project:javax/swing/plaf/nimbus/ScrollBarTrackPainter.java':[
88,
],
'project:javax/swing/plaf/nimbus/ScrollPanePainter.java':[
75,
],
'project:javax/swing/plaf/nimbus/SeparatorPainter.java':[
72,
],
'project:javax/swing/plaf/nimbus/SliderPainter.java':[
70,
],
'project:javax/swing/plaf/nimbus/SliderTrackPainter.java':[
83,
],
'project:javax/swing/plaf/nimbus/SpinnerPainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/SplitPanePainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/TabbedPanePainter.java':[
75,
],
'project:javax/swing/plaf/nimbus/TableEditorPainter.java':[
76,
],
'project:javax/swing/plaf/nimbus/TableHeaderPainter.java':[
74,
],
'project:javax/swing/plaf/nimbus/ToolTipPainter.java':[
73,
],
'project:javax/swing/plaf/nimbus/TreeCellEditorPainter.java':[
76,
],
'project:javax/swing/text/ComponentView.java':[
219,
],
'project:javax/swing/text/FieldView.java':[
125,
209,
],
'project:javax/swing/text/IconView.java':[
111,
],
'project:javax/swing/text/JTextComponent.java':[
740,
4504,
],
'project:javax/swing/text/PasswordView.java':[
219,
],
'project:javax/swing/text/html/ImageView.java':[
499,
],
'project:javax/swing/text/html/StyleSheet.java':[
1974,
],
'project:javax/swing/text/html/parser/DTD.java':[
180,
],
'project:javax/swing/text/html/parser/Parser.java':[
1213,
1562,
],
}
